.travis.yml

```diff
@@ -8,6 +8,8 @@ sudo: false
 before_install:
    - export PATH=$HOME/gopath/bin:$PATH
 
+install: "pip install --user -r requirements.txt"
```

Comment:
@sbinet Ah okay. Actually, I am not familiar with yum install, So I need to find how to install cffi with yum. :)